app/models/event.rb

```diff
@@ -106,4 +106,7 @@ def hide_cover_fees?
 		nonprofit.hide_cover_fees? || misc_event_info&.hide_cover_fees_option
 	end
 
+	def custom_get_tickets_button_label
```

Comment:
So this is a super tiny request but could this just be "get_tickets_button_label"? It's not necessarily custom.

Reply:
My thinking was that it was custom because the default is "Get Tickets". Do you want me to change the attribute name too?

Reply:
The default is "Get Ticket" but if you call this method and no custom one has been set, then you'll get the default. So on event it's a just the label of the the get tickets button.

Reply:
Ahh, gotcha :)